Tests/UIKit/UIViewControllerTests.swift

```diff
@@ -28,11 +28,11 @@ class UIViewControllerTests: XCTestCase {
         let viewController = UIViewController()
         _viewController = viewController
         
-        viewController.rex_dismissModally.signal.observeNext { _ in
+        viewController.rex_dismissAnimated.signal.observeNext { _ in
             expectation.fulfill()
         }
                 
-        viewController.rex_dismissModally <~ SignalProducer(value: (true, nil))
+        viewController.rex_dismissAnimated <~ SignalProducer(value: (true, nil))
```

Comment:
Maybe we can be explicit about the parameters of the tuple (sometimes tests are used as usage reference).